### ISSUE TITLE
issue with conda installation

### ISSUE THREAD

**Issue by ptitle** [dportik/SuperCRUNCH#11]
I may simply be doing something wrong as I am new to conda, but from within the SuperCrunch directory, I run

`conda create -f supercrunch-conda-env.yml`

But I get:

```
usage: conda create [-h] [--clone ENV] (-n ENVIRONMENT | -p PATH) [-c CHANNEL] [--use-local]
                    [--override-channels] [--repodata-fn REPODATA_FNS] [--experimental {jlap,lock}]
                    [--strict-channel-priority] [--no-channel-priority] [--no-deps | --only-deps] [--no-pin]
                    [--copy] [-C] [-k] [--offline] [--json] [-v] [-q] [-d] [-y] [--download-only]
                    [--show-channel-urls] [--file FILE] [--no-default-packages]
                    [--solver {classic,libmamba} | --experimental-solver {classic,libmamba}] [--dev]
                    [package_spec ...]
conda create: error: one of the arguments -n/--name -p/--prefix is required
```
I can see that a name is specified in the yml file, so I'm not sure why this error is being returned. 

Any suggestions? Thanks!

**Comment by dportik** [comment]
Hi @ptitle ,
That does seem strange.

Can you try the following instead:

```
conda create -n supercrunch -f supercrunch-conda-env.yml
```

Let me know if that works and I can update the docs. It might be a conda version thing, and that the command is outdated.

**Comment by ptitle** [comment]
I gave that a try, and this is what I got... This is on macOS 13.5, for what it's worth.

```
➜  SuperCRUNCH git:(master) conda create -n supercrunch -f supercrunch-conda-env.yml
Collecting package metadata (current_repodata.json): / DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
\ DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/r/noarch/current_repodata.json HTTP/1.1" 304 0
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/r/osx-arm64/current_repodata.json HTTP/1.1" 304 0
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/main/osx-arm64/current_repodata.json HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/main/noarch/current_repodata.json HTTP/1.1" 304 0
done
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
Collecting package metadata (repodata.json): - DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): repo.anaconda.com:443
- DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/r/osx-arm64/repodata.json HTTP/1.1" 304 0
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/main/noarch/repodata.json HTTP/1.1" 304 0
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/main/osx-arm64/repodata.json HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:https://repo.anaconda.com:443 "GET /pkgs/r/noarch/repodata.json HTTP/1.1" 304 0
done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - supercrunch-conda-env.yml

Current channels:

  - https://repo.anaconda.com/pkgs/main/osx-arm64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-arm64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

**Comment by ptitle** [comment]
I found that running `conda env create -f supercrunch-conda-env.yml` worked, but then I ran into a bunch of issues where certain software was not available. I think this may be related to this machine having the Apple M2 chip, and the library of programs is either not as complete, or there is some Mac-specific place I need to point to. 

I had better luck taking the non-conda route, as I could find just about all required programs via homebrew.
